apps/condo/domains/ticket/access/TicketComment.js

```diff
@@ -48,8 +48,8 @@ async function canReadTicketComments ({ authentication: { item: user } }) {
         ticket: {
             organization: {
                 OR: [
-                    queryOrganizationEmployeeFor(user.id),
```

Comment:
Should there be a `canReadTicketComments` permission here?
In function `checkManageCommentAccess` we use `canManageTicketComments`

Reply:
I think it's redundant to add `canReadTicketComments` because we can not be able to manage it from ui.
If we need it, we’ll add